nil/tests/cli/cli_test.go

```diff
@@ -626,6 +626,14 @@ func (s *SuiteCliExec) TestCliCometa() {
 	})
 }
 
+func (s *SuiteCliExec) TestCallCliTxPool() {
```

Comment:
Move this test into rpc test suite. There is no connection with CLI here

Reply:
https://github.com/NilFoundation/nil/blob/main/nil/tests/rpc_service/rpc_service_test.go